samples/sample-spring-boot/src/test/java/com/nike/wingtips/springbootsample/componenttest/VerifySampleEndpointsComponentTest.java

```diff
@@ -269,8 +463,24 @@ public void verify_async_error_endpoint_traced_correctly(boolean upstreamSendsSp
                 .log().all()
                 .extract();
 
-        assertThat(response.statusCode()).isEqualTo(503);
-        verifySingleSpanCompletedAndReturnedInResponse(response, SLEEP_TIME_MILLIS, upstreamSpanInfo.getLeft());
+        assertThat(response.statusCode()).isEqualTo(500);
+        Span completedSpan =
+            verifySingleSpanCompletedAndReturnedInResponse(response, SLEEP_TIME_MILLIS, upstreamSpanInfo.getLeft());
+        // Springboot sets the org.springframework.web.servlet.HandlerMapping.bestMatchingPattern request attribute
```

Comment:
@adriancole another subtle glitchy thing I found that you might want to be aware of, or help me out if you know how to better extract the HTTP route in this particular case. See the inline code comment here. This seems to only affect springboot as far as I can tell - I couldn't reproduce this on a Spring MVC sample. 🤷‍♂️